packages/orchestration/src/exos/chain-hub.js

```diff
@@ -223,38 +293,48 @@ export const makeChainHub = (agoricNames, vowTools) => {
      * @param {IBCConnectionInfo} connectionInfo
      */
     registerConnection(chainId1, chainId2, connectionInfo) {
-      const key = connectionKey(chainId1, chainId2);
-      connectionInfos.init(key, connectionInfo);
+      const [key, normalized] = normalizeConnectionInfo(
```

Comment:
This I am a little confused by - is it necessary for the `hotnewchain` scenario, since we can't be certain they will give us `noramlizedConnInfo`? _It would seem `orchestration-proposal.js` has everything normalized already at this point._

I wonder if it's better to throw if `chainId1` > `chainId2` so the user knows how they need to specify the perspective of `connectionInfo`. Alternatively, maybe a parameter name change would do the trick.

Reply:
> I wonder if it's better to throw if `chainId1` > `chainId2` so the user knows how they need to specify the perspective of `connectionInfo`.

That would not be reliable. 

> Alternatively, maybe a parameter name change would do the trick.

Agree, I'll improve the docs.